ui/raidboss/data/06-ew/raid/p2n.ts

```diff
@@ -180,13 +174,7 @@ const triggerSet: TriggerSet<Data> = {
       netRegexDe: NetRegexes.startsUsing({ id: '6807', source: 'Hippokampos', capture: false }),
       netRegexFr: NetRegexes.startsUsing({ id: '6807', source: 'Hippokampos', capture: false }),
       netRegexJa: NetRegexes.startsUsing({ id: '6807', source: 'ヒッポカムポス', capture: false }),
-      alertText: (_data, _matches, output) => output.text!(),
-      outputStrings: {
-        text: {
-          en: 'Knockback--Stay on grid',
-          de: 'Rückstoß--Bleib auf den Gittern stehen',
-        },
-      },
+      response: Responses.knockback(),
```

Comment:
Similarly, I think "don't get knocked into bad stuff" felt fairly obvious even the first time through, and I think it'd be better to not complicate this.  I don't think we'd say "knockback--don't get hit into spikes" for p4n.

Reply:
Arms length / surecast is up during these casts so just kb is fine.